### DISCUSSION TITLE
How do I go to a specific path without clicks?

### DISCUSSION THREAD

**Discussion by gerroon** [spacedriveapp/spacedrive#1782] [Community Support]
Hi

Lets say I want to go  to c:\users\users\appdata\local, I want to do it with autocomplete in  a path bar or something, without constantly clicking, like in the Win explorer bar? Possible?

**Comment by ketonik** [comment]
Yeah, it's be nice to have functionality similar to BASH terminal, start typing path, hit TAB key and it auto presents available paths by means of filtering,

so example
type 
c:\us hit TAB it shows list \users then again us c:\users\users\ then again type app hit TAB \appdata\ etc etc

**Comment by utkubakir** [comment]
This is planned and will hopefully be out soon.
